src/docs/user/references/JobPlannerServiceReference.adoc

```diff
@@ -21,6 +21,23 @@ One can see the cron expression "0 23 0 ? * MON,TUE,WED,THU,FRI *", which follow
 <<_job_planner_cron_expression_syntax, Quartz Cron Expression Syntax>> section. The cron expression in this example
  executes 23 minutes past every hour, on working days (Monday to Friday).
 
+==== Inclusion Calendar
+
+The purpose of the inclusion calendar is to use ICS file to launch workflows during events. For each events a pre-defined action will be executed an will define when the workflow will be executed.
```

Comment:
The purpose of the inclusion calendars section is to specify workflow launching policies during calendar events. For instance, automatically launch a specific workflow/action at event start.